test-data/fake-repo-with-config/aws-codeguru-reviewer.yml

```diff
@@ -0,0 +1,7 @@
+version: 1.0
+
+# Sample YAML file
+# https://docs.aws.amazon.com/codeguru/latest/reviewer-ug/welcome.html.
```

Comment:
As above, I couldn't find anything related at this URL.

Reply:
will fix